plugin/pkg/shoot/validator/admission.go

```diff
@@ -519,9 +519,16 @@ func validateKubernetesVersionConstraints(constraints []core.ExpirableVersion, s
 		}
 
 		if getLatestPatchVersion {
```

Comment:
Unrelated to your change, just saw it: Can you fix line 511 and convert both times to UTC?